department-of-veterans-affairs/vets-api
# Add Terms of Use check to SessionRefresher

## Summary

- Check ToU when sessions attempts to refresh 

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#67750

## How to test
- start `vets-website`
- start `vets-api`
- in browser `#1` login with SiS and accept terms
- in browser `#2` login with SiS
- in browser `#1` navigate to [localhost:3001/terms-of-use](localhost:3001/terms-of-use) and Decline.
- You should be logged out of browser `#1`.
- In browser `#2` you should be logged out once you access token expires (~5 mins)

## What areas of the site does it impact?
Terms of Use, Sign-in Service

## Acceptance criteria
